source/tyr/migrations/versions/105823db902c_add_autocomplete_backend.py

```diff
@@ -0,0 +1,22 @@
+"""Add autocomplete_backend to Instance
+
+Revision ID: 105823db902c
+Revises: 483639f1f00
+Create Date: 2018-08-30 07:57:48.630448
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '105823db902c'
```

Comment:
i'm not sure, but should you change the revision? because i've just added a new one :p

Reply:
The first merged win, I'm first :D